bootstrap/mod/src/main/java/org/geysermc/geyser/platform/mod/GeyserModUpdateListener.java

```diff
@@ -25,17 +25,16 @@
 
 package org.geysermc.geyser.platform.mod;
 
-import net.minecraft.commands.CommandSourceStack;
 import net.minecraft.world.entity.player.Player;
-import org.geysermc.geyser.Constants;
-import org.geysermc.geyser.platform.mod.command.ModCommandSender;
+import org.geysermc.geyser.Permissions;
+import org.geysermc.geyser.platform.mod.command.ModCommandSource;
 import org.geysermc.geyser.util.VersionCheckUtils;
 
 public final class GeyserModUpdateListener {
     public static void onPlayReady(Player player) {
-        CommandSourceStack stack = player.createCommandSourceStack();
-        if (GeyserModBootstrap.getInstance().hasPermission(stack, Constants.UPDATE_PERMISSION, 2)) {
-            VersionCheckUtils.checkForGeyserUpdate(() -> new ModCommandSender(stack));
+        ModCommandSource source = new ModCommandSource(player.createCommandSourceStack());
+        if (source.hasPermission(Permissions.CHECK_UPDATE)) {
+            VersionCheckUtils.checkForGeyserUpdate(() -> source);
```

Comment:
revert but use `source.getServer().getOperatorUserPermissionLevel()` instead of `2`

Reply:
unable